kura/org.eclipse.kura.asset.provider/src/main/java/org/eclipse/kura/asset/provider/BaseAsset.java

```diff
@@ -542,41 +507,45 @@ private AssetRecord createAssetRecordFromDriverRecord(DriverRecord driverRecord)
 
     /** {@inheritDoc} */
     @Override
-    public List<AssetRecord> read(final List<Long> channelIds) throws KuraException {
+    public List<AssetRecord> read(final Set<String> channelNames) throws KuraException {
         requireNonNull(this.driver, message.driverNonNull());
```

Comment:
Yeah you are right and that's why in my previous review, I suggested you to remove it.